Header header
OCRObject[] qr_tags
OCRObject[] giant_locations
